redisinsight/ui/src/services/theme.ts

```diff
@@ -24,7 +24,7 @@ class ThemeService {
 
     document.adoptedStyleSheets = [sheet]
 
-    localStorageService.set(BrowserStorageItem.theme, actualTheme)
```

Comment:
I've tested both desktop + docker image, and IMO this is the only change we need. Can you:
- verify it suits your needs
- leave just this change

Reply:
The problem if we keep just this change is that my settings will not be saved anywhere so if I open new Redisinsight window (eg. chrome incognito) it will be opened with default theme ie. `System Theme`. Added videos for before and after including the (complete) changes.

before - 

https://github.com/user-attachments/assets/108e48f6-fac7-463f-937f-8b56a0b60c07

at 7s I changed the theme to `Dark`, at 15s the theme is again light (`System`)

after - 

https://github.com/user-attachments/assets/8ecec791-f46f-4156-88c8-e6988f4045d2

at 5s I changed the theme to `Dark`, at 15s the theme is `Dark` (fetched from server)

Thats why other changes are necessary for complete solution.

Reply:
@pawelangelow shall I mark this conversation as resolved?

Reply:
I'll double-check it. Just to confirm, we want to cover the following scenarios:

- When the system theme changes and the application follows it, the app should respond accordingly (e.g., from dark to light mode).

- When the user changes the theme manually, the choice should be persistently saved.

Reply:
correct